portal/src/components/v2/TextField/TextField.tsx

```diff
@@ -92,7 +86,35 @@ export function TextField({
             <Icon icon={iconEnd} />
           </RadixTextField.Slot>
         ) : null}
-      </RadixTextField.Root>
+      </TextInput>
     </FormField>
   );
 }
+
+export function TextInput({
```

Comment:
We should move TextInput to its own directory. Add a story for TextInput

Reply:
Same as ImageInput, it is not a component in the storybook design, so I do not add a story.
I can still move it to a new directory.

Reply:
The conclusion is move TextInput to its own file. It can live without a story.